src/app/modules/dashboard/pages/dashboard/components/cpu-stats/cpu-stats.component.ts

```diff
@@ -0,0 +1,116 @@
+import {Component, OnInit, OnDestroy, ViewChild} from '@angular/core';
+import {Color, BaseChartDirective} from 'ng2-charts';
+import {timer, BehaviorSubject, Subject, Subscription} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
+import {ToastrService} from 'ngx-toastr';
+
+import {DashboardMetricsService} from '../../../../services/dashboard-metrics.service';
+
+@Component({
+    selector: 'app-cpu-stats-component',
+    templateUrl: './cpu-stats.component.html',
+    styleUrls: ['./cpu-stats.component.scss']
+})
+export class CpuStatsComponent implements OnInit, OnDestroy {
+    @ViewChild(BaseChartDirective, {static: false})
+    cpuChart: BaseChartDirective;
+
+    loading$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(true);
+
+    public cpuLineChartData: Array<any> = [0];
+    public cpuLineChartLabels: Array<any> = [''];
+
+    public cpuLineChartOptions: any = {
+        responsive: true,
+        scales: {
+            xAxes: [
+                {
+                    gridLines: {
+                        display: false
+                    }
+                }
+            ],
+            yAxes: [
+                {
+                    id: 'y-0',
+                    position: 'left',
+                    ticks: {
+                        beginAtZero: true,
+                        max: 100,
+                        min: 0
+                    },
+                    gridLines: {
+                        display: false
+                    }
+                }
+            ]
+        }
+    };
+
+    public cpuLineChartColors: Color[] = [
+        {
+            backgroundColor: 'rgba(0,0,0,0)',
+            borderColor: '#2196F3',
+            pointBackgroundColor: 'rgba(148,159,177,1)',
+            pointBorderColor: '#fff',
+            pointHoverBackgroundColor: '#fff',
+            pointHoverBorderColor: 'rgba(148,159,177,0.8)'
+        }
+    ];
+
+    public cpuLineChartLegend = false;
+    public cpuLineChartType = 'line';
```

Comment:
The `ng2-charts` element expects the `chartType` to be of a specific type: 
```suggestion
    import * as chartJs from 'chart.js';
    ...
    public cpuLineChartType: chartJs.ChartType = 'line';
```
Please make this change to the other components as well.